priv/base/runner

```diff
@@ -90,13 +90,13 @@ This is the primary script for controlling the $SCRIPT node.
         Attaches to the console of a $SCRIPT node running in the background
         using an Erlang remote shell, giving access to the Erlang shell and
         runtime messages. Prints "Node is not running!" when the node cannot
```

Comment:
"Node is not running!" is now replaced with "Node did not respond to ping!" per #199